services/QuillLMS/client/app/bundles/Staff/components/attributesManager/topics/topicBox.tsx

```diff
@@ -116,7 +117,12 @@ const TopicBox = ({ originalTopic, levelThreeTopics, saveTopicChanges, closeTopi
   }
 
   function renderLevels() {
-    const dropdown = topic.level === 2 ? renderDropdownInput() : null
+    let dropdown
```

Comment:
You could abstract this into a shared function - it's declared twice; here and newTopicBox.tsx